ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-pim-sparse-mode.cfg

```diff
@@ -15,6 +15,9 @@ router pim sparse-mode
       rp address 10.238.1.161 239.12.12.21/32 priority 20
       rp address 10.238.1.161 access-list RP_ACL priority 20
       rp address 10.238.1.161 access-list RP_ACL2 priority 20
+      rp address 10.238.1.161 239.12.12.12/32 priority 20
+      rp address 10.238.1.161 access-list RP_ACL priority 20
+      rp address 10.238.1.161 access-list RP_ACL2 priority 20
```

Comment:
Seems you have exact duplicate examples in your input. Can you check and we should handle that case that if other params also same then it should allow duplicate

Reply:
Removed duplicate test.